### PR TITLE
docs: Update _index.md

### PR DESCRIPTION
**Pull request by martinblascop**
**Description**
Update v0.6.3 docs

**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/v0.6.3/concepts/_index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Concepts associated with this role are described below.
 Karpenter is designed to run on a node in your Kubernetes cluster.
 As part of the installation process, you need credentials from the underlying cloud provider to allow nodes to be started up and added to the cluster as they are needed.
 
-[Getting Started with Karpenter on AWS]{{< ref "../getting-started" >}})
+[Getting Started with Karpenter on AWS]({{< ref "../getting-started" >}})
```
Comment (FernandoMiguel) [inline website/content/en/v0.6.3/concepts/_index.md:27]: those this only impact v0.6.3?

**Comment by jonathan-innis** [comment]
@martinblascop Thanks for your contribution! Can you do a find a replace for this issue across the docs? As @FernandoMiguel mentioned, this affects more than `v0.6.3` so if you could fix this across versions that would be great!

**Comment by FernandoMiguel** [comment]
I did a very quick search and this was the only place for this.
It also shows in 0.16.3 but different implementation.

**Comment by jonathan-innis** [comment]
> I did a very quick search and this was the only place for this. It also shows in 0.16.3 but different implementation.

Huh, I also found it throughout other docs...from v0.6.3 and below essentially

**Review by ellistarn — APPROVED** [review]

**Pull request merged**